.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-3.0.3
```

Comment:
You need to follow your decisions. If you added that file to `.gitignore` it shouldn't go to the repo then.

Reply:
I will remove it